tools/epoch-sync/src/cli.rs

```diff
@@ -0,0 +1,257 @@
+use anyhow::Context;
+use clap;
+use near_chain::{ChainStore, ChainStoreAccess, ChainUpdate, DoomslugThresholdMode};
+use near_epoch_manager::shard_tracker::{ShardTracker, TrackedConfig};
+use near_epoch_manager::EpochManager;
+use near_primitives::block::BlockHeader;
+use near_primitives::borsh::BorshDeserialize;
+use near_primitives::epoch_manager::block_info::BlockInfo;
+use near_primitives::epoch_manager::AGGREGATOR_KEY;
+use near_primitives::hash::CryptoHash;
+use near_store::{checkpoint_hot_storage_and_cleanup_columns, DBCol, NodeStorage};
+use nearcore::NightshadeRuntime;
+use std::collections::{HashMap, HashSet};
+use std::path::{Path, PathBuf};
+
+#[derive(clap::Parser)]
+pub struct EpochSyncCommand {
+    #[clap(subcommand)]
+    subcmd: SubCommand,
+}
+
+#[derive(clap::Parser)]
+#[clap(subcommand_required = true, arg_required_else_help = true)]
+enum SubCommand {
+    ValidateEpochSyncInfo,
+}
+
+impl EpochSyncCommand {
+    pub fn run(self, home_dir: &Path) -> anyhow::Result<()> {
+        let mut near_config = EpochSyncCommand::create_snapshot(home_dir)?;
```

Comment:
Nit:

```suggestion
        let mut near_config = Self::create_snapshot(home_dir)?;
```

(this is slightly easier to maintain in case the type becomes generic for example, and is in general just shorter)